.github/scripts/check-react-native-log.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+
+echo "cd build-system-tests/mega-apps/${MEGA_APP_NAME}"
+cd build-system-tests/mega-apps/${MEGA_APP_NAME}
+
+# Define the time of seconds to wait the command
+if [ "$PLATFORM" = "ios" ]; then
+  TIME_TO_WAIT=300
+else
+  TIME_TO_WAIT=200
+fi
+
+# Define the color codes
+BLUE_BOLD="\033[1;36m"
+GREEN_BOLD="\033[1;32m"
+RED_BOLD="\033[1;31m"
+YELLOW_BOLD="\033[1;33m"
+RESET="\033[0m"
+
+# Sleep for the specified time
+echo -e "${BLUE_BOLD}Sleep for $TIME_TO_WAIT second...${RESET}"
+sleep $TIME_TO_WAIT
+
+# Check if the log file only has one line
+if [[ $(wc -l <"$LOG_FILE") -eq 1 && $(head -n 1 "$LOG_FILE") == "info Starting logkitty" ]]; then
+  echo -e "${RED_BOLD}Failed to get the logging messages. Please increase TIME_TO_WAIT.${RESET}"
```

Comment:
Is the only reason this would print due to time out issues? If so we should log an error that states that explicitly, if not then removing "Please increase TIME_TO_WAIT." would make sense since the failure is not guaranteed to be time out related

Reply:
If`TIME_TO_WAIT` is not long enough, we would only see the initial line of log. Is there anything else needs to be added in the error message to make it more explicit?

Reply:
I guess the question is whether the if clause evaluates to true if and only if `TIME_TO_WAIT` is not long enough. If there are other scenarios that will evaluate the if clause to true, "Please increate `TIME_TO_WAIT` line might be inaccurate and so we can consider dropping it.